frontend/components/top_nav/SiteTopNav/_styles.scss

```diff
@@ -81,13 +80,13 @@
 }
 
 .logo {
-  height: 48px;
```

Comment:
This needs to be here or else custom images will only be half size which might be larger or smaller than 48px.
I also noticed icons with larger widths are also not fixed with this:

Example adding a custom image using Organization settings > Organization avatar URL:
<img width="807" alt="Screenshot 2023-01-27 at 4 34 10 PM" src="https://user-images.githubusercontent.com/71795832/215205086-757f0861-3d72-4fca-b545-acb07f8ca6a5.png">

Reply:
Thanks for pointing that out! Will fix.